src/ssb_project_cli/ssb_project/app.py

```diff
@@ -312,7 +399,9 @@ def delete():
                 delete_cmds += [["poetry", "env", "remove", venv.split(" ")[0]]]
 
     for cmd in delete_cmds:
-        deletion = subprocess.run(cmd, capture_output=True)
+        deletion = subprocess.run(  # noqa: S603 no untrusted input
+            cmd, capture_output=True
+        )  # noqa: S603 no untrusted input
```

Comment:
It shouldn't be necessary to have this comment twice

Reply:
It does not, fixed 👍